From 1c301bcaa56fee40d5c52ddc3a6c3f03b350073c Mon Sep 17 00:00:00 2001
From: Felipe Contreras <felipe.contreras@gmail.com>
Date: Wed, 3 May 2023 17:23:49 -0600
Subject: [PATCH 1400/2871] doc: doc-diff: specify date

Earlier we changed the manual page formatting machinery to use the
dates from the commit the documentation source was taken from,
instead of the date the manual page was produced.  When "doc-diff"
compares two commits from different dates, the different dates from
the two commits would result in unnecessary differences in the
output because of the change.

Compensate by setting a fixed date when "doc-diff" formats the pages
to be compared to work around this issue.

Signed-off-by: Felipe Contreras <felipe.contreras@gmail.com>
Signed-off-by: Junio C Hamano <gitster@pobox.com>
---
 Documentation/doc-diff | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/doc-diff b/Documentation/doc-diff
index 1694300e50..554a78a12d 100755
--- a/Documentation/doc-diff
+++ b/Documentation/doc-diff
@@ -153,6 +153,7 @@ render_tree () {
 		make -j$parallel -C "$tmp/worktree" \
 			$makemanflags \
 			GIT_VERSION=omitted \
+			GIT_DATE=1970-01-01 \
 			SOURCE_DATE_EPOCH=0 \
 			DESTDIR="$tmp/installed/$dname+" \
 			install-man &&
-- 
2.39.3 (Apple Git-145)

